cluster-capi-kind/README.md

```diff
@@ -0,0 +1,5 @@
+# cluster
+
+## Description
+
+This package provides a blueprint manifest to deploy a kind cluster using cluster api
```

Comment:
Another thing to document are the host dependencies like, `docker` and the `cni plugins` folder location in the host.